analysis/cpp/src/make-histograms-loop.cpp

```diff
@@ -115,22 +140,45 @@ int main(int argc, char* argv[]) {
         // h_vtzzz->Fill(pv_zpos * 1e3);
 
         // loop over the MC particles
-        // for(size_t iparticle = 0; iparticle < n_mc_particles_in_event; ++iparticle) {
-        //     int pdgid = mcpdg[iparticle];
-
-        //     // skip things we don't care about
-        //     if(std::abs(pdgid) > 2000) continue;
-
-        //     h_mcpdg->Fill(pdgid);
-
-        //     // MC particle momentum of this particle
-        //     double px = mc_px[iparticle];
-        //     double py = mc_px[iparticle];
-        //     double transverse_momentum = sqrt(px*px + py*py); // GeV
-        //     if(transverse_momentum > 1.0) {
-        //         h_mcpdg_ptcut->Fill(pdgid);
-        //     }
-        // } // loop over iparticle
+        float mu_vtx_x_pos = 0;
+        float antimu_vtx_x_pos = 0;
```

Comment:
`mu_vtx_x_pos` and `antimu_vtx_x_pos` will store the muon and antimuon MC vertex x-positions, respectively.